gstaichi/runtime/cpu/jit_cpu.cpp

```diff
@@ -131,10 +131,21 @@ class JITSessionCPU : public JITSession {
         mangle_(es_, this->dl_),
         module_counter_(0),
         memory_manager_(nullptr) {
+#if !(defined(__APPLE__) && defined(__aarch64__))
+    // On ELF-based targets LLVM's ORC JIT expects the object layer to claim
+    // responsibility for every symbol emitted by the module. Without this the
+    // resolver may encounter symbols that were not pre-registered in the
+    // materialization responsibility set and abort with
+    // "Resolving symbol outside this responsibility set" (observed on
+    // manylinux aarch64).
+    object_layer_.setOverrideObjectFlagsWithResponsibilityFlags(true);
+    object_layer_.setAutoClaimResponsibilityForObjectSymbols(true);
+#else
```

Comment:
I still don't understand this part. I would avoid altering the current behaviour as much as possible. So the condition should be at best `#if defined(__aarch64__) && !defined(__APPLE__)`. Moreover, Apple using `__arm64__`, not `__aarch64__`, so in the end it should just be: `#if defined(__aarch64__):`

Actually, the condition `defined(__APPLE__) && defined(__aarch64__)` may be possible on manylinux ARM container running on MacOS host, eventually, but I did not checked that. I don't really think it makes any sense still.

Reply:
Still I think it should just be `if linux and aarch64`, arm64 is MacOS specific, not Apple ARM-based CPU arch.

Reply:
(I think it's already "if linux and aarch64" now?)